skactiveml/classifier/_wrapper.py

```diff
@@ -322,3 +350,874 @@ def __getattr__(self, item):
+
+        # Check class prior.
+        self.class_prior_ = check_class_prior(
+            self.class_prior, len(self.classes_)
+        )
+
+        # Check whether metric is available.
+        if self.metric not in KERNEL_PARAMS and not callable(
+                self.metric
+        ):
+            raise ValueError(
+                "The parameter 'metric' must be callable or "
+                "in {}".format(KERNEL_PARAMS.keys())
+            )
+
+        # Check number of neighbors which must be a positive integer.
+        if self.n_neighbors is not None:
+            check_scalar(
+                self.n_neighbors,
+                name="n_neighbors",
+                min_val=1,
+                target_type=int,
+            )
+
+        # Ensure that metric_dict is a Python dictionary.
+        self.metric_dict_ = (
+            self.metric_dict if self.metric_dict is not None else {}
+        )
+        if not isinstance(self.metric_dict_, dict):
+            raise TypeError("'metric_dict' must be a Python dictionary.")
+
+        if not hasattr(self, "frequency_estimator_"):
+            self.frequency_estimator_ = SubSampleEstimator(
+                ParzenWindowClassifier(
+                    n_neighbors=self.n_neighbors,
+                    metric=self.metric,
+                    metric_dict=self.metric_dict,
+                    missing_label=np.nan,
+                    classes=[0],
+                    random_state=self.random_state_.randint(2 ** 31 - 1),
+                ),
+                max_fit_len=self.frequency_max_fit_len,
+                replacing_strategy=self.frequency_replacing_strategy,
+                only_labled=True,
+                classes=[0],
+                missing_label=np.nan,
+            )
+
+        return X, y, sample_weight
+
+    def __getattr__(self, item):
+        if "estimator_" in self.__dict__ and hasattr(self.estimator_, item):
+            return getattr(self.estimator_, item)
+        else:
+            raise AttributeError(f"{item} does not exist")
+
+
+class SubSampleEstimator(SkactivemlClassifier, MetaEstimatorMixin):
+    """SubSampleEstimator
+
+    Implementation of a wrapper class for SkactivemlClassifier such that the 
+    number of training samples can be limited. Furthermore, saves X, y and
+    sample_weight, enabling the use of a partial fit for any classifier. The
+    class offers two updating strategies, i.e., replacing the oldes samples
+    (sliding window) or replacing samples randomly.
+
+    Parameters
+    ----------
+    estimator : sklearn.base.SkactivemlClassifier
+        The wrapped classifier.
+    classes : array-like of shape (n_classes,), default=None
+        Holds the label for each class. If none, the classes are determined
+        during the fit.
+    missing_label : scalar or string or np.nan or None, default=np.nan
+        Value to represent a missing label.
+    cost_matrix : array-like of shape (n_classes, n_classes)
```

Comment:
redundant. similar problem with sklearn-wrapper? what if sklearn clf already supports cost_matrix?

Reply:
at minimum: ensure that everything is the same